blog.html

```diff
@@ -71,6 +71,12 @@
               </p>
               <section class="light">
                 <ul style="list-style-type: none">
+                <li class="leading-block">
+                    <span class="blog-post-date">21 July, 2022</span>
```

Comment:
date will have to change (as you mentioned on discord). Just flagging for visibility.